.github/workflows/publish_sdist_to_pypi.yml

```diff
@@ -77,15 +83,26 @@ jobs:
         tag: ${{ github.ref }}
         overwrite: true
         file_glob: true
-    - name: Publish package distributions to PyPI
+    - name: Check dist contents
+      run: twine check dist/*
+    - name: Publish package dist to internal PyPI
       if: ${{ steps.check-tag.outputs.match != '' }}
-      uses: pypa/gh-action-pypi-publish@v1.7.1
-      with:
-        password: ${{ secrets.PYPI_API_TOKEN }}
-    - name: Publish package distributions to TestPyPI
-      if: ${{ steps.check-tag.outputs.match == '' }}
-      uses: pypa/gh-action-pypi-publish@v1.7.1
-      with:
-        password: ${{ secrets.TESTPYPI_API_TOKEN }}
-        repository-url: https://test.pypi.org/legacy/
-        verbose: true
+      run: |
+        export no_proxy=${{ secrets.PYPI_HOST }}
+        export REPOSITORY_URL=http://${{ secrets.PYPI_HOST }}:${{ secrets.PYPI_PORT }}
+        twine upload --verbose --repository-url $REPOSITORY_URL dist/* -u ${{ secrets.PYPI_USER }} -p ${{ secrets.PYPI_PASSWORD }}
+    # - name: Publish package distributions to TestPyPI
+    #   if: ${{ steps.check-tag.outputs.match == '' }}
+    #   uses: pypa/gh-action-pypi-publish@v1.7.1
+    #   with:
+    #     password: ${{ secrets.TESTPYPI_API_TOKEN }}
+    #     repository-url: https://test.pypi.org/legacy/
+    #     verbose: true
```

Comment:
Why did you remove the existing publish pipeline (to public PYPI)?

Reply:
due to the action pypa/gh-action-pypi-publish need to have 'docker' inside of the ci-runner machine but did not have that. I will update those part to use 'twine' next time.

Reply:
and as I've told you in IM, publish to the public PyPI will be done manually